README.md

```diff
@@ -1,23 +1,40 @@
 # project-template
 Project template based on gulp, jade and postcss.
 ## Install
-```
-git clone https://github.com/yakhinvadim/project-template
-cd project-template
-npm install
-```
-then you can run tasks, described below
+
+Action | Command
+:--- | :---
+clone repository | `git clone https://github.com/yakhinvadim/project-template your-project-name`
+enter project folder | `cd project-template`
+install gulp globally | `npm install -g gulp`
```

Comment:
extract this as separate step before all the commands in this section, because its not required to it everytime